python/pyarrow/tests/test_flight.py

```diff
@@ -1545,6 +1578,48 @@ def test_roundtrip_errors():
         with pytest.raises(flight.FlightInternalError, match=".*foo.*"):
             list(client.list_flights())
 
+        with pytest.raises(flight.FlightInternalError, match=".*foo.*"):
+            writer, reader = client.do_put(flight.FlightDescriptor.for_command("internal"))
```

Comment:
do_put also needs the schema (so pass table.schema as the second argument)

Reply:
ok